# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/gtk/GtkCompositorWidget.h
# Commit: 2bf4f54a3077
# Full Hash: 2bf4f54a307746331f89e76ea9be8daaf9732886
# Author: stransky <stransky@redhat.com>
# Date: 2024-12-17 05:09:10
# Regressor Bug: 1934497
# File Overlap Count: 1
# Description:
#   Bug 1934497 [Wayland] Update GtkCompositorWidget for WaylandSurface r=emilio
#   
#   - Replace MozContainerSurfaceLock with WaylandSurfaceLock
#   - Temporary disable GtkCompositorWidget::GetNativeLayerRoot()
#   
# ==============================================================================

diff -r 87613cc32aa4 -r 2bf4f54a3077 widget/gtk/GtkCompositorWidget.h
--- a/widget/gtk/GtkCompositorWidget.h	Mon Dec 16 20:12:16 2024 +0000
+++ b/widget/gtk/GtkCompositorWidget.h	Mon Dec 16 20:12:17 2024 +0000
@@ -11,7 +11,7 @@
 #include "mozilla/widget/CompositorWidget.h"
 #include "WindowSurfaceProvider.h"
 #include "mozilla/UniquePtr.h"
-#include "MozContainerSurfaceLock.h"
+#include "WaylandSurfaceLock.h"
 
 class nsIWidget;
 class nsWindow;
@@ -96,7 +96,7 @@
   void NotifyClientSizeChanged(const LayoutDeviceIntSize& aClientSize) override;
   GtkCompositorWidget* AsGtkCompositorWidget() override { return this; }
 
-  UniquePtr<MozContainerSurfaceLock> LockSurface();
+  UniquePtr<WaylandSurfaceLock> LockSurface();
 
  private:
 #if defined(MOZ_WAYLAND)
